public/includes/calculate-recommendation.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
Kindly implement the same function in Javascript and use in `script.js`. Making this a PHP file is an overkill :smirk